EDITORS.md

```diff
@@ -78,3 +79,26 @@ require("lspconfig").ruby_ls.setup({
   end,
 })
 ```
+
+## Sublime Text LSP
+
+[LSP for Sublime Text](https://github.com/sublimelsp/LSP) has built-in support for [Solargraph](https://lsp.sublimetext.io/language_servers/#solargraph) and [Sorbet](https://lsp.sublimetext.io/language_servers/#sorbet)
+
+To add ruby-lsp support, add the following configuration to your LSP client configuration:
+
+```json
+"clients": {
+  "ruby-lsp": {
+    "enabled": true,
+    "command": [
+      "ruby-lsp"
+    ],
+    "selector": "source.ruby | text.html.ruby | text.html.erb | text.html.rails",
```

Comment:
We don't support any file extensions other than Ruby yet. For example, we haven't worked on adding `erb` support.

So I think we can limit the selector to `.rb` Ruby files only for now.